src/components/CButton2/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {TouchableOpacity, Text, StyleSheet} from 'react-native';
+import {colors, textStyles} from "assets";
+import {scale} from "device";
+
+interface Props {
+    text?: string,
+    disabled?: boolean,
+    style?: any,
+    icon?: any,
+    textStyle?: any,
+    type?: 'default' | 'line'
+}
+
+const defaultProps = {
+    onPress: () => console.log('KIEM TRA'),
+};
+
+const CButton2 = ({disabled = false, style, icon, text, textStyle, type = 'default', ...rest}: Props) => {
```

Comment:
Should we choose another meaningful name for this component? CButton2 looks hard to understand for another developers. Thanks.

Reply:
Is CTextButton ok? If not, please suggest a name.  Because it already exist a basic CButton. I added CButton2 that follow your design. Thanks
<img width="868" alt="Screen Shot 2022-01-17 at 16 01 16" src="https://user-images.githubusercontent.com/34257375/149739347-adfc6f92-d586-4edf-a8a9-e7db95c7220e.png">

Reply:
hi @djhahe , any suggestions?

Reply:
CTextButton sounds good